[has fix] error: 'std::filesystem' has not been declared

## summary
Several errors when running `make`, like this:
```
pprepair/IOWorker.cpp:1242:20: error: 'path' is not a member of 'std::filesystem'
 1242 |   std::filesystem::path extension = std::filesystem::path(file).extension();
      |                    ^~~~
```

## fix available
#57

--- a/IOWorker.cpp
+++ b/IOWorker.cpp
@@ -20,6 +20,7 @@
  */
 
 #include "IOWorker.h"
+#include <filesystem>
 
 IOWorker::IOWorker() {
   startingSearchFace = Triangulation::Face_handle();

